recipes/frugen/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get, rmdir, apply_conandata_patches, export_conandata_patches
+import os
+
+class FrugenConan(ConanFile):
+    name = "frugen"
+    description = "IPMI FRU Information generator / editor tool and library"
+    license = ("Apache-2.0", "GPL-2.0-or-later")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://codeberg.org/IPMITool/frugen"
+    topics = ("hardware", "ipmi", "fru")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_json": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_json": True
+    }
+    implements = ["auto_shared_fpic"]
+
+    exports_sources = "CMakeLists.txt", "*.c", "*.h"
```

Comment:
```suggestion
```

The recipe is already brining the sources from the release

Reply:
Actually, I just have no clear understanding, what it suppose to do )